book/src/installation.md

```diff
@@ -10,6 +10,19 @@ The easiest way to install both is with [`rustup.rs`](https://rustup.rs/).
 On Windows, you will also need a recent version of [Visual Studio](https://visualstudio.microsoft.com/downloads/),
 installed with the "Desktop Development With C++" Workloads option.
 
+If you want to enable `estark-polygon` feature, you also need the following
+runtime dependencies installed on the system:
+- `gcc`
+- `nlohmann-json3-dev`
+
+Yes, gcc and headers at runtime, this is not a typo.
```

Comment:
wdym `gcc and headers at runtime`? I think this sentence is unnecessary. If we want to explain why they're needed we could, but this sentence doesn't do that